src/containers/CreateDoc/pdf.ts

```diff
@@ -0,0 +1,28 @@
+import PdfMake from 'pdfmake/build/pdfmake';
+
+import { TDocumentDefinitions } from 'types';
+import { TCreatedPdf } from 'types';
+
+interface PdfDefinitions {
+    [pdfDefinition: string]: TDocumentDefinitions
+}
+
+const pdfDefinitions: PdfDefinitions = {
+    default : {
+        content: 'Content',
+        header: 'Header',
+        footer: 'Footer'
+    },
+    sOne: {
+        content: 'Content',
+        header: 'Header',
+        footer: 'Footer',
+        background: 'background'
+    }
+}
+
+function createPdf(pdfType: string = 'default'): TCreatedPdf {
+    return PdfMake.createPdf(pdfDefinitions[pdfType]);
```

Comment:
Are `open` and `download` function inside the obj return of this function ? 

---------

Anyway, we should try to send the inner text to this module, from the createDoc `index` so we start wiring some things up between the modules.

Reply:
Fair enough